### PR TITLE
refactor(homepage): adds new homepage image and imposes text on top

### PR DESCRIPTION
**Pull request by eve-horne**
# Changes introduced
Changed homepage image to match new screenshots found on CSE-330.
Updates homepage text container to lie on top of background image.
Added borders to the homepage text container.

## Related issue(s):

- CSE-330

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [x] Added new stories for created/modified components (if applicable)
- [x] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [x] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

**Review by tnoelcke — DISMISSED** [review]
When running this on a slightly smaller screen this yielded some odd results. We may want to remove the home page from the generic page layout so we can get a bit more predictable scoring behavior.

**Comment by puregarlic** [comment]
@tnoelcke, can you take another look at these changes?

File: src/pages/Home/index.tsx [hunk 1]
```diff
@@ -10,36 +9,47 @@ const importedLogo = homeLogo as string;
...
-      <h1 className="passage-title">About CSE</h1>
+      <h1 className="passage-title">Welcome to Content Spec Explorer!</h1>
       <div className="passage-content">
-        Ipsum aut accusamus quod omnis. Ex fugit eaque aut quia dolorum quaerat. Consequuntur
-        occaecati id voluptatibus illo enim. Velit eveniet consequuntur recusandae laboriosam eos
-        maiores. Illo earum voluptatem et praesentium quia et. Non est et cumque ullam eveniet quas
-        est molestiae. Et eveniet atque laboriosam provident et aut recusandae. Eos occaecati aut
-        laboriosam facere facilis qui. Maxime et omnis ipsum et minus. Eligendi repudiandae aut
-        quasi.
+        The CSE website provides easy access to a host of Smarter Balanced’s test development
```
Comment (tnoelcke) [inline src/pages/Home/index.tsx:16]: Thank you for getting this text integrated.

**Review by tnoelcke — APPROVED** [review]
Looks good.

**Pull request merged**